Sources/Client/WebSocket/ConnectionState.swift

```diff
@@ -37,8 +37,11 @@ public enum ConnectionState: Equatable {
         switch (lhs, rhs) {
```

Comment:
If you made `ClientError` equatable (the way you compare it here in the switch), this custom implementation could go away.